get-random-throw.js

```diff
@@ -0,0 +1,48 @@
+export function getRandomThrow() {
+    const randomNumber = Math.ceil(Math.random() * 3);
+
+    let computerChoice;
+
+    if (randomNumber === 1) {
+        computerChoice = 'rock';
+    } else if (randomNumber === 2) {
+        computerChoice = 'paper';
+    } else if (randomNumber === 3) {
+        computerChoice = 'scissors';
+    }
+
+
+    return computerChoice;
+}
+
+
+export function userWins(userGuess, computerChoice) {
+ }   if (userGuess === 'rock' && computerChoice === 'rock') {
```

Comment:
You have an extra curly brace in this function. Also, please read the directions. 
Returns one of the following values:
'draw' - the throws are identical
'win' - the player's throw beats the computer throw
'lose' - the computer's throw beats the player's throw